php/libraries/NDB_Page.class.inc

```diff
@@ -721,6 +722,7 @@ class NDB_Page implements RequestHandlerInterface
     {
         switch($this->format) {
         case 'json':
+            // NOTE this function doesn't actually exist but seems to work?!
```

Comment:
if it works, why add the function ?

Reply:
I don't remember the nuances of this 🤷‍♂ I'll delete it